git-rebase-via-merge.sh

```diff
@@ -6,41 +6,45 @@
 # Copyright (c) 2022 Baur Atanov
 #
 
-default_base_branch="origin/develop"
+default_base_branch="origin/main"
```

Comment:
I can revert this change if desired; since [GitHub changed the default branch to `main`](https://github.blog/changelog/2020-10-01-the-default-branch-for-newly-created-repositories-is-now-main/) I figured it'd make sense.